js/components/utils.js

```diff
@@ -579,12 +579,20 @@ var Utils = {
   * @param {Object} thumbnails - metadata object containing information about thumbnails
   * @param {Number} hoverTime - time value to find thumbnail for
   * @param {Number} duration - duration of the video
+  * @param {Boolean} isVideoVr - if video is vr
   * @returns {Object} object that contains URL and index of requested thumbnail
   */
-  findThumbnail: function(thumbnails, hoverTime, duration) {
+  findThumbnail: function(thumbnails, hoverTime, duration, isVideoVr) {
     var timeSlices = thumbnails.data.available_time_slices;
     var width = thumbnails.data.available_widths[0]; //choosing the lowest size
 
+    if (isVideoVr) {
+      // it is necessary to take bigger image for showing part of the image
```

Comment:
Why bigger image?
- Player will have considerable performance downgrade due to downloading bigger size image
- I do not want the thumbnail size to blow up (or shrink)
- Has this been discussed with Steve? I disagree with the usage of bigger images.